package.json

```diff
@@ -192,7 +192,8 @@
     "vue-eslint-parser": "^7.1.0",
     "webpack": "^4.44.2",
     "webpack-manifest-plugin": "^2.2.0",
-    "xml2js": "^0.4.23"
+    "xml2js": "^0.4.23",
+    "eslint-plugin-security-node": "1.0.12"
```

Comment:
Why this isn't sorted? We should have a pre-commit hook to sort the package.json file

Reply:
You can manually trigger it with `npx sort-package-json`